packages/tailwindcss/src/theme.ts

```diff
@@ -69,6 +69,10 @@ export class Theme {
     return ((this.values.get(key)?.options ?? 0) & ThemeOptions.DEFAULT) === ThemeOptions.DEFAULT
   }
 
+  getOptions(key: string) {
+    return this.values.get(key)?.options ?? 0
```

Comment:
This might be better for readability:
```suggestion
    return this.values.get(key)?.options ?? ThemeOptions.NONE
```

Reply:
Updated it there and in a few other spots